src/Okta.Sdk/Internal/PayloadHandler.cs

```diff
@@ -0,0 +1,139 @@
+﻿// <copyright file="PayloadHandler.cs" company="Okta, Inc">
+// Copyright (c) 2020 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Net.Http;
+
+namespace Okta.Sdk.Internal
+{
+    /// <summary>
+    /// A class responsible for providing body and message content.
+    /// </summary>
+    public abstract class PayloadHandler : IPayloadHandler
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PayloadHandler"/> class.
+        /// </summary>
+        public PayloadHandler()
+        {
+            Serializer = new DefaultSerializer();
+        }
+
+        static PayloadHandler()
```

Comment:
Missing access modifier?

Reply:
Default is private, I'll add it in explicitly.